cpp/src/Slice/Parser.cpp

```diff
@@ -732,6 +732,12 @@ namespace
 
 CommentPtr
 Slice::Contained::parseComment(bool stripMarkup) const
+{
+    return parseComment(_comment, stripMarkup);
+}
+
+CommentPtr
+Slice::Contained::parseComment(const string& text, bool stripMarkup) const
```

Comment:
I added this overload to be able to preprocess the coment before parsing. This allow for language specific preprocessing.